charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.117.3
+version: 0.118.0
```

Comment:
@jenrik we should not change anything in this pr outside the operator chart. Can you pls delete this as it seems stale.